test/JDBC/expected/13_4__preparation__BABEL-3556-vu-prepare.out

```diff
@@ -0,0 +1,176 @@
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: value "-922337203685478.5808" is out of range for type fixeddecimal)~~
+
+
+INSERT INTO babel_3556_vu_prepare_t1 VALUES(922337203685478.5807);
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: value "922337203685478.5807" is out of range for type fixeddecimal)~~
+
+
+INSERT INTO babel_3556_vu_prepare_t1 VALUES(-922337203685478.5808);
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: value "-922337203685478.5808" is out of range for type fixeddecimal)~~
+
+-- integral
+SELECT CAST(922337203685478 AS fixeddecimal);
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: data type fixeddecimal is not supported yet)~~
+
+
+SELECT CAST(-922337203685478 AS fixeddecimal);
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: data type fixeddecimal is not supported yet)~~
+
+
+SELECT CAST(1232422322334223423121 AS fixeddecimal);
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: value "1232422322334223423121" is out of range for type fixeddecimal)~~
+
+
+INSERT INTO babel_3556_vu_prepare_t1 VALUES(922337203685478);
+GO
+~~ROW COUNT: 1~~
+
+
+INSERT INTO babel_3556_vu_prepare_t1 VALUES(-922337203685478);
+GO
+~~ROW COUNT: 1~~
+
+
+INSERT INTO babel_3556_vu_prepare_t1 VALUES(1232422322334223423121);
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: value "1232422322334223423121" is out of range for type fixeddecimal)~~
+
+
+
+-- To test out of range values for money
+CREATE TABLE babel_3556_vu_prepare_t3(a money);
+GO
+
+CREATE TABLE babel_3556_vu_prepare_t4(a money);
+GO
+
+-- decimal
+SELECT CAST($922337203685478.5807 AS money);
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: value "922337203685478.5807" is out of range for type fixeddecimal)~~
+
+
+SELECT CAST(₧-922337203685478.5808 AS money);
+GO
+
+INSERT INTO babel_3556_vu_prepare_t3 VALUES($922337203685478.5807);
+GO
+
+INSERT INTO babel_3556_vu_prepare_t3 VALUES(₧-922337203685478.5808);
+GO
+-- integral
+SELECT CAST(₧922337203685478 AS money);
+GO
+
+SELECT CAST($-922337203685478 AS money);
+GO
+~~START~~
+money
+922337203685477.1616
+~~END~~
+
+
+SELECT CAST($1232422322334223423121 AS money);
+GO
+
+INSERT INTO babel_3556_vu_prepare_t3 VALUES($922337203685478);
+GO
+~~ROW COUNT: 1~~
+
+
+INSERT INTO babel_3556_vu_prepare_t3 VALUES(₧-922337203685478);
+GO
+
+INSERT INTO babel_3556_vu_prepare_t3 VALUES(1232422322334223423121);
+GO
+
+-- To test out of range values for smallmoney
+CREATE TABLE babel_3556_vu_prepare_t5(a smallmoney);
+GO
+
+CREATE TABLE babel_3556_vu_prepare_t6(a smallmoney);
+GO
+
+-- decimal
+SELECT CAST($214748.3648 AS smallmoney);
+GO
+~~START~~
+smallmoney
+~~END~~
+
+
+SELECT CAST(₧-214748.3649 AS smallmoney);
+GO
+
+INSERT INTO babel_3556_vu_prepare_t5 VALUES($214748.3648);
+GO
+
+INSERT INTO babel_3556_vu_prepare_t5 VALUES(₧-214748.3649);
+GO
+
+-- integer
+SELECT CAST($214749 AS smallmoney);
+GO
+~~START~~
+smallmoney
+~~END~~
+
+
+SELECT CAST(₧-214749 AS smallmoney);
+GO
```

Comment:
no result?

Reply:
Earlier `multi-byte currency symbol` wasn't supported, it has been supported recently in BABEL-2636, I will replace these with `single-byte ($)` as testcases for that has been done already